### PR TITLE
Display created_at if solution published_at equals V2 launch date

### PR DESCRIPTION
**Pull request by PatrickMcSweeny**
Fix for exercism/exercism#4073

### CODE REVIEW

**Comment by PatrickMcSweeny** [comment]
Not sure why [this test](https://github.com/exercism/website/blob/master/test/services/select_suggested_solutions_for_mentor_test.rb#L260) is failing. All the tests pass when I run them locally.

File: app/models/solution.rb [hunk 1]
```diff
@@ -44,6 +44,14 @@ def self.not_legacy
     where("solutions.created_at >= ?", Exercism::V2_MIGRATED_AT)
   end
 
+  def display_published_at
```
Comment (iHiD) [inline app/models/solution.rb]: Thanks for solving this. Could you do this using the tertiary operator to keep things a bit slimmer pls?

**Comment by PatrickMcSweeny** [comment]
Done

**Comment by iHiD** [comment]
Great. Thanks :)

**Pull request merged**